p_periodic.json

```diff
@@ -22,7 +21,7 @@
     "4.16 to 4.17 upgrade":"periodic-ci-openshift-multiarch-master-nightly-4.17-upgrade-from-nightly-4.16-ocp-ovn-remote-libvirt-multi-p-p",
     "4.17 libvirt": "periodic-ci-openshift-multiarch-master-nightly-4.17-ocp-e2e-ovn-remote-libvirt-ppc64le",
```

Comment:
Add entry for this also- https://prow.ci.openshift.org/job-history/gs/origin-ci-test/logs/periodic-ci-openshift-multiarch-master-nightly-4.17-ocp-e2e-ovn-remote-libvirt-multi-p-p

Reply:
ok